compiler/aarch64/codegen/OMRMemoryReference.cpp

```diff
@@ -750,6 +750,12 @@ static bool isImm12OffsetInstruction(uint32_t enc)
    return ((enc & 0x3b200000) == 0x39000000);
    }
 
+/* stp/ldp GPR */
+static bool isImm7OffsetGPRInstruction(uint32_t enc)
+   {
+   return ((enc & 0x7e000000) == 0x28000000);
```

Comment:
I think you cannot use `ldpsw` (pair of signed word) instructions with this condition.
- ldpswoff: 0x69400000
- ldpswpost: 0x68C00000
- ldpswpre: 0x69C00000

Only support standard `stp` and `ldp` instructions for GPRs this time?

Reply:
Thanks for pointing it out. I relaxed the check to support `ldpsw`.